explore/src/main/scala/explore/targeteditor/AsterismEditor.scala

```diff
@@ -106,14 +113,9 @@ object AsterismEditor {
                 )
               ),
               onSelected = Reuse
-                .always(_ match {
+                .by((props.obsIds, props.asterism))(_ match {
```

Comment:
Should `selectedTargetId` be part of the reusability conditions?

Reply:
Yeah. It probably should. It wasn't being used for reuse before, and I think it works because `selectedTargetId` is only set, not modified. But it is probably a recipe for weird behavior in the future. I'll add it to the reusability.